extensions/wikia/WikiFactory/tests/WikiFactoryTest.php

```diff
@@ -114,43 +122,96 @@ public function testGetLocalEnvURLDataProvider() {
 		return [
 			[
 				'env' => WIKIA_ENV_PREVIEW,
+				'forcedEnv' => null,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "forcedEnv" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)